### ISSUE TITLE
Different Sizes for Cards.

### ISSUE THREAD

**Issue by MunchkinMan** [kellyelton/OCTGN#325]
TL;DR: The ability to define more then one size for cards within an octgn set/game, Reducing the clickable area.

To be more specific. Lets say If the predefined area of the most common card used for a game is 50x50, but there are a few cards that happen to be 50X100; As of now you can create a png that would be the defined 50x50 and clear space top and bottom, but the card would be really small. Being able to properly define card sizes for games that have different sized cards could give it a more realistic feel.

Simplistic example would be for this picture. note the size difference in the normal cards and dungeon cards: http://cf.geekdo-images.com/images/pic1226569_md.jpg

another example would be Ninja Burger. the cards are either tall or wide. http://cf.geekdo-images.com/images/pic53656_md.jpg
This would leave more clickable space on the top or bottom.

Being able to define the clickable area/size for cards would be very benificial for board games as well.
How about one last example. The board game Zombies. http://cf.geekdo-images.com/images/pic162574_md.jpg
Looking at this picture you can see the 4 card sizes needed for this game (board pieces, player/zombie tokes, heart/bullet tokens and action cards). the largest being the actual board tiles and the smallest being the bullet tokens.
If the largest defined size would be the board pieces, the clickable area of every other card/token would be as large as a piece of board. that would make moving pieces around dificult and would make the game harder to play.

I'm not sure if this will ever be possible but i'll put it back out there for consideration.
